### ISSUE TITLE
[Feature] Filter toxic words/sentences from a given text 

### ISSUE THREAD

**Issue by Bytes-Explorer** [data-prep-kit/data-prep-kit#383]
### Search before asking

- [X] I searched the [issues](https://github.com/IBM/data-prep-lab/issues) and found no similar issues.

### Component

Transforms/Other

### Feature

Implement a new feature to detect and eliminate toxic words or sentences from a given text. This functionality should work on every row of a parquet file, where every row contains one document. The output should be True or False, and this should be added as an output column along with span of the text where toxic words are detected. 

This can be added as a new transform for text/NLP data. One can refer to code quality module as a reference for how filters have been applied for code data. 

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by daw3rd** [comment]
Is this a duplicate of #389

**Comment by Bytes-Explorer** [comment]
No, it is not. There is some overlap in the labels of hate, toxicity but they have been used as separate labels as well in the literature.
